cmd/run/run.go

```diff
@@ -410,6 +413,20 @@ func run(_ *cobra.Command, _ []string) {
 	}
 }
 
+func readCertificate(ctx context.Context, certPath, keyPath string) (*certwatcher.CertWatcher, error) {
+	watcher, err := certwatcher.New(certPath, keyPath)
```

Comment:
Do we want to assert whether certPath and keyPath are specified prior to calling the certWatcher.New?

Reply:
Done. Use `os.Stat` to confirm the file beforehand